MobileWallet/Common/Theme.swift

```diff
@@ -132,6 +137,12 @@ struct Fonts: Loopable {
     //Splash
     let splashTitleLabel = UIFont(name: "AvenirLTStd-Black", size: 30.0)
 
+    //SplashCreatingWallet
+    let firstLabel = UIFont(name: "AvenirLTStd-Black", size: 18.0)
```

Comment:
Need more descriptive var names in a theme file. There could be many first labels. Something like  `splashCreateWalletLabel1`